packages/form-builder/src/App.tsx

```diff
@@ -22,6 +22,8 @@ const initialState = {
         registration: 'none',
         goalType: 'amount',
         designId: 'classic',
+        heading: __('Support Our Cause'),
+        description: __('Help our organization by donating today! Dnations go to making a difference for our cause.'),
```

Comment:
It's great we're using the translation function but i'm pretty sure if you don't pass `'give'` as the second argument it wont actually do anything since it needs to locate the textdomain.